src/main/script/indexMongoCollections.cmd

```diff
@@ -0,0 +1,5 @@
+@REM Place this script on mongoDB installation directory. It runs an instance of mongo daemon and a .js script that adds indexes for xlimeres. Change --dbpath arguments if needed.
```

Comment:
Don't think it's necessary to place in mongoDB install directory. See below: I wouldn't start the mongod. Add warning that if you execute this against a database that is already populated (and being written to), all operations will be blocked until the collections have been indexed (which can take a while, so some write operations may fail).